src/utils/arguments.ts

```diff
@@ -0,0 +1,3 @@
+export const enforceNoArgumentsMessage = (commandName: string): string => {
+  return `\`${commandName.toLowerCase()}\` doesn't take any arguments.`;
```

Comment:
If I do `.balance 234` or whatever, it'll say dont give args. Why not run anyways?

Reply:
Oh this was discussed in https://github.com/uwcsc/codeybot/pull/176#discussion_r823344516, in which the conclusion seems to be to enforce 0 arguments?

Reply:
I have now removed 0 argument enforcement.